geoportal/requirements.txt

```diff
@@ -1,3 +1,4 @@
 c2cgeoportal_commons==2.3.5.25
 c2cgeoportal_geoportal==2.3.5.25
 c2cgeoportal_admin==2.3.5.25
+cairocffi==0.9.0
```

Comment:
CairoSVG doesn't work on the current version because something doens't work with python 2
There is version of CairoSVG for python 3, we should use theme instead of using a previous version.
Plus, it should be good to keep only one "requierement", see: https://github.com/Geoportail-Luxembourg/geoportailv3/blob/master/setup.py#L26 (Looks missing on our 2.3)

Reply:
OK, done